packages/api/public/spec/index.yml

```diff
@@ -6,13 +6,10 @@ info:
     The frontend can be accessed at [https://gloss-translation.vercel.app]().
   version: 0.0.1
 servers:
-  - url: https://gloss-translation-api.vercel.app/api
-    description: Production server
-  - url: http://localhost:4300/api
-    description: Local server
```

Comment:
I remove this in favor of just a path. By default the explorer will use the URL of the server it is running on. I was getting annoyed with forgetting to change to the local server when testing